tests/system/python/e2e/test_e2e_notification_service_with_plugins.py

```diff
@@ -271,31 +281,28 @@ def enable_notification(self, foglamp_url, cat_name, is_enabled=True):
         r = conn.getresponse()
         assert 200 == r.status
 
-    def test_sent_and_receive_notification(self, foglamp_url, start_south, wait_time):
+    def sent_and_receive_notification(self, foglamp_url, start_south, wait_time):
         data = {"name": "Test4", "description": "Test4_Notification", "rule": NOTIFY_RULE, "channel": NOTIFY_PLUGIN,
-                "enabled": False, "notification_type": "retriggered"}
+                "enabled": True, "notification_type": "retriggered"}
         name = data['name']
         _add_notification_instance(foglamp_url, data)
         self.configure_rule_with_latest_eval_type(foglamp_url, name)
         self.enable_notification(foglamp_url, name)
-
-        time.sleep(wait_time)
         self.ingest_readings_from_fogbench(foglamp_url, wait_time)
-
+        pause_for_x_seconds(x=5)
         _verify_audit_log_entry(foglamp_url, '/foglamp/audit?source=NTFSN', name=name)
 
 
 class TestStartStopNotificationService:
-
     def test_shutdown_service_with_schedule_disable(self, foglamp_url, disable_schedule, wait_time):
         disable_schedule(foglamp_url, SERVICE_NAME)
-
         _verify_service(foglamp_url, status='shutdown')
-        time.sleep(wait_time)
+        pause_for_x_seconds(x=5)
         _verify_audit_log_entry(foglamp_url, '/foglamp/audit?source=NTFSD', name=SERVICE_NAME)
 
+    @pytest.mark.skip(reason="FOGL-2640 - Frequent errors in restarting Notification service")
     def test_restart_notification_service(self, foglamp_url, enable_schedule):
         enable_schedule(foglamp_url, SERVICE_NAME)
-
+        pause_for_x_seconds(x=5)
         _verify_service(foglamp_url, status='running')
-        _verify_audit_log_entry(foglamp_url, '/foglamp/audit?source=NTFST', name=SERVICE_NAME)
+        _verify_audit_log_entry(foglamp_url, '/foglamp/audit?source=NTFST', name=SERVICE_NAME, count=2)
```

Comment:
We should have explanation for / clear test data. why 2 are expected. and what are those should be asserted.